spec/index.html

```diff
@@ -547,21 +548,37 @@ <h2>RDF Graphs</h2>
 
   <p>An <dfn>RDF graph</dfn> is a set of <a>RDF triples</a>.</p>
 
-  <p>An RDF triple is said to be <dfn id="dfn-asserted-triple" data-lt="asserted|asserted triple">asserted</dfn> in an RDF graph if it is an element of the RDF graph.</p>
-
   <section id="section-triples">
     <h3>Triples</h3>
 
-    <p>An <dfn data-local-lt="triple">RDF triple</dfn> (usually called "triple")
-      is a 3-tuple (|s|, |p|, |o|) with the following characteristics:</p>
+    <p>An <dfn data-local-lt="triple" class="export">RDF triple</dfn>
+     (often simply called "triple")
+     is a 3-tuple that is defined inductively as follows:</p>
 
     <ul>
-      <li>|s| is an <a>IRI</a> or a <a>blank node</a>.</li>
-      <li>|p| is an <a>IRI</a>.</li>
-      <li>|o| is an <a>IRI</a>, a <a>blank node</a>, a <a>literal</a>,
-        or a <a>triple term</a>.</li>
+      <li>
+        If |s| is an [=IRI=] or a [=blank node=],
+        |p| is an [=IRI=], and
+        |o| is an [=IRI=], a [=blank node=], or a [=literal=],
+        then (|s|, |p|, |o|) is an [=RDF triple=].
+      </li>
+
+      <li>
+        If |s| is an [=IRI=] or a [=blank node=],
+        |p| is an [=IRI=], and
+        |o| is an [=RDF triple=],
```

Comment:
Why not?
```suggestion
        |o| is an [=IRI=], a [=blank node=], a [=literal=], or an [=RDF triple=],
```

Reply:
Because all these other options are already covered by the first point.

Reply:
Well, yes, my point was "why two `<li>`? why not fold them together?" 

There's no difference between them, except the nested `[=RDF triple=]`. maybe I should have replaced the entirety of the two `<li>` with one `<li>`, instead of replacing the last line of the first `<li>` through the last line of the second `<li>` with one last line that combines the two last lines....

Reply:
Because "[a recursive definition must always have base cases, cases that satisfy the definition _without_ being defined in terms of the definition itself](https://en.m.wikipedia.org/wiki/Recursive_definition#Form_of_recursive_definitions)." In the given definition, the first bullet point is that base case.

Reply:
what @hartig said. Otherwise, it wouldn't be a well-formed inductive/recursive definition.